python/MoltresXS.py

```diff
@@ -0,0 +1,280 @@
+"""
+     Python Function that takes homoginezed cross sections and prepares them for MOLTRES
+
+"""
+
+import io
+import sys
+import numpy as np
+import h5py
+from pyne import serpent
+
+
+class scale_xs:
```

Comment:
I strongly recommend that classes should have more than just one super-long init function. There are a lot of important things that are enabled by functions and the DRY (don't repeat yourself) principle:

- debugging
- flexibility
- testing
- readability
- extensibility